lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -92,6 +92,149 @@ html.video-fullscreen {
       display: inline;
     }
   }
+
```

Comment:
I know this PR specifically has merged, but @chrisndodge one suggestion here might be to further nest these buttons under something that will make sure it only impacts the messages area for proctoring / timed exams. I'm not familiar with the existing html but I imagine an existing class could be used to address this reduced scoping. I've cc'ed @frrrances here since she's taking care of the Solutions PR work, but since I was tagged earlier I thought I'd make sure to but this note in. 

Thanks!